VALIDATION.md

```diff
@@ -3,14 +3,21 @@
 ### Manual Steps
 
 #### Icons
-1. Validate a beta package integrates into the showcase demo.
+
+1. Validate a beta package integrates into the react demo.
 
 #### Icons MUI
-1. Validate a beta package integrates into the showcase demo.
+
+1. Validate a beta package integrates into the react demo.
 
 #### React Native Vector Icons
-1. Validate a beta package integrates into the showcase demo.
 
+1. Validate a beta package integrates into the react demo.
```

Comment:
You can't use RN vector icons in a react project. Change this to say something like a new RN project from the CLI instead.